plugins/ruby-foreman-azure/debian/gem.list

```diff
@@ -1,8 +1,8 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_azure-1.0.1.gem"
+GEMS="https://rubygems.org/downloads/foreman_azure-1.1.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/fog-azure-0.0.2.gem"
-GEMS="$GEMS https://rubygems.org/downloads/azure-0.7.5.gem"
-GEMS="$GEMS https://rubygems.org/downloads/azure-core-0.1.1.gem"
+GEMS="$GEMS https://rubygems.org/downloads/azure-0.7.7.gem"
+GEMS="$GEMS https://rubygems.org/downloads/azure-core-0.1.5.gem"
 GEMS="$GEMS https://rubygems.org/downloads/faraday_middleware-0.10.0.gem"
```

Comment:
Updating faraday_middleware to 0.10.1 should permit faraday 0.10.0, fixing the build failure.